src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const count = 40;
+  const discount = count * days;
+
+  if (days >= 3 && days <= 6) {
+    return discount - 20;
+  }
+
+  if (days >= 7) {
+    return discount - 50;
+  }
+
+  return count * 2;
```

Comment:
Prefer using 'const' over 'let' where possible to avoid unintentional changes. In this case, 'days' can be declared as 'const' if it is not meant to be reassigned, as per the checklist item 3.